docs/en/start-v2/kubernetes/helm.md

```diff
@@ -0,0 +1,76 @@
+---
+sidebar_position: 4
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Set Up with Helm
+
+This section provides a quick guide to use SeaTunnel with Helm.
+
+## Prerequisites
+
+We assume that you have one local installation as follow:
+
+- [docker](https://docs.docker.com/)
+- [kubernetes](https://kubernetes.io/)
+- [helm](https://helm.sh/docs/intro/quickstart/)
+
+So that the `kubectl` and `helm` commands are available on your local system.
+
+Take kubernetes [minikube](https://minikube.sigs.k8s.io/docs/start/) as an example, you can start a cluster with the following command:
+
+```bash
+minikube start --kubernetes-version=v1.23.3
+```
+
+## Install
+
+Install with default settings.
+```bash
+# Choose the corresponding version yourself
+export VERSION=2.3.9
+helm pull oci://registry-1.docker.io/apache/seatunnel-helm --version ${VERSION}
+tar -xvf seatunnel-helm-${VERSION}.tgz
+cd seatunnel-helm
+helm install seatunnel .
+```
+
+If you want to install under another namespace.
+First, update `value.yaml`
+```
+## change
+seatunnel.default.svc.cluster.local
+## to
+seatunnel.<your namespace >.svc.cluster.local
+
+helm install seatunnel . -n <your namespace>
+```
+
+## Submit Job
+
+The default config doesn't enable ingress, so you need forward the master restapi.
+```bash
+kubectl port-forward -n default svc/seatunnel-master 5801:5801
+```
+Then you can access restapi with "http://127.0.0.1/5801/"
+
+If you want to use ingress, update `value.yaml`
```

Comment:
We should use new rest api port to submit job. Please refer https://github.com/apache/seatunnel/blob/dev/docs/en/seatunnel-engine/rest-api-v2.md#overview